dumpling/README.md

```diff
@@ -1,16 +1,12 @@
 🥟 Dumpling
 ============
 
-[![Build Status](https://travis-ci.org/pingcap/dumpling.svg?branch=master)](https://travis-ci.org/pingcap/dumpling)
-[![codecov](https://codecov.io/gh/pingcap/dumpling/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/dumpling)
-[![API Docs](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/pingcap/dumpling)
-[![Go Report Card](https://goreportcard.com/badge/github.com/pingcap/dumpling)](https://goreportcard.com/report/github.com/pingcap/dumpling)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fpingcap%2Fdumpling.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fpingcap%2Fdumpling?ref=badge_shield)
+[![API Docs](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/pingcap/tidb/dumpling)
 
-**Dumpling** is a tool and a Go library for creating SQL dump from a MySQL-compatible database.
+**Dumpling** is a tool and a Go library for creating an SQL dump from a MySQL-compatible database.
 It is intended to replace `mysqldump` and `mydumper` when targeting TiDB.
 
-You may read the [design document](https://github.com/pingcap/community/blob/master/rfc/2019-12-06-dumpling.md), [English user guide](docs/en/user-guide.md) and [中文使用手册](docs/cn/user-guide.md) for details.
+You may read the [design document](https://github.com/pingcap/community/blob/master/archive/misc/rfc/2019-12-06-dumpling.md), [English user guide](docs/en/user-guide.md) and [中文使用手册](docs/cn/user-guide.md) for details. The end-user documentation is available here: [en](https://docs.pingcap.com/tidb/stable/dumpling-overview), [zh](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview), [ja](https://docs.pingcap.com/ja/tidb/stable/dumpling-overview).
```

Comment:
I'd like one source of information, and in this line we have "user guide" vs "end-user documentation". Maybe we can remove the "user guide"? I guess it's less maintained.

Reply:
I've updated this PR to remove that and https://github.com/pingcap/docs/pull/19890 to merge the info into pingcap/docs.

cc @qiancai